src/drivers/distance_sensor/broadcom/afbrs50/AFBRS50.cpp

```diff
@@ -206,6 +212,9 @@ void AFBRS50::Run()
 	// backup schedule
 	ScheduleDelayed(100_ms);
 
+	UpdateMode();
+	get_mode();
```

Comment:
This is printing to the console every iteration?

Reply:
Was for debugging, I'll take that out.